tuf/api/exceptions.py

```diff
@@ -11,6 +11,7 @@
 #### Repository errors ####
 
 # pylint: disable=unused-import
+# ruff: disable unused-inport
 from securesystemslib.exceptions import StorageError
```

Comment:
the way to handle this with ruff is 
```suggestion
from securesystemslib.exceptions import StorageError  # noqa: F401
```

then yuou can remove the file level ignore in pyproject.toml

Reply:
actually I'm sure the name works as well and might be more readable 
```suggestion
from securesystemslib.exceptions import StorageError  # noqa: unused-import
```